e2e/bitrise.yml

```diff
@@ -207,6 +217,16 @@ workflows:
     - _run_test_without_building
 
   test_test_plan_enabled_project_test_plan_specified:
+    steps:
+    - script:
+        run_if: "{{getenv \"BITRISEIO_STACK_ID\" | ne \"osx-xcode-14.3.x-edge\"}}"
```

Comment:
Do you know why these tests are not compatible with the `osx-xcode-14.3.x-edge` stack?

Reply:
Yesterday simulators were so slow on this image, that tests were failing. Since that, new images have been built, so reenabled this stack and it seems to be working: https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/51/commits/b43e4e7c7ea7d9dac258e26da98fbeaa9cedc1cd

Reply:
Now the osx-xcode-15-edge stack seems to have the same issue, so it is excluded: https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/51/commits/03b30efcc1883521ce53dcaa111b1ccd7262e3c7
All the stacks are passing, but not this edge. There is some flakiness in the test plan related tests or on the edge stacks.

I would merge my changes and handle this flakiness separately.